.github/workflows/test.yaml

```diff
@@ -209,8 +209,16 @@ jobs:
         # install secboot again
         ${{ github.workspace }}/bin/govendor remove github.com/snapcore/secboot
         ${{ github.workspace }}/bin/govendor remove +unused
-        # we don't need coverage anymore
-        SKIP_COVERAGE=1 SKIP_DIRTY_CHECK=1 GO_BUILD_TAGS=nosecboot ./run-checks --unit
+        SKIP_DIRTY_CHECK=1 GO_BUILD_TAGS=nosecboot ./run-checks --unit
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
+      with:
+        # token: ${{ secrets.CODECOV_TOKEN }} # needed ?
+        fail_ci_if_error: true
```

Comment:
Why is this needed?

Reply:
This makes the action fail the unit-tests action if there is an error uploading, otherwise it will just silently fail and not report back to us the coverage. I think this is good, since it means if we ever have problems uploading then we will find out about it via failing actions on PR's.